client/src/pages/Homepage.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+import PostInfo from "../components/PostInfo";
+
+const Homepage = () => {
+  const [posts, setPosts] = useState([]);
+
+  const getPosts = async () => {
+    try {
+      const response = await fetch("http://localhost:8080/api/v1/posts");
+      const {
+        data: { posts },
+      } = await response.json();
+      setPosts(posts);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getPosts();
+  }, []);
+
+  return (
+    <div>
```

Comment:
Might be cool to have a "loading" state that starts off `true`, and to display a "loading..." (or maybe a spinner) while the `useEffect()` is running to fetch your data.